internal/kafka/command_mirror.go

```diff
@@ -7,18 +7,19 @@ import (
 )
 
 type mirrorOut struct {
-	LinkName                 string `human:"Link Name" serialized:"link_name"`
-	MirrorTopicName          string `human:"Mirror Topic Name" serialized:"mirror_topic_name"`
-	SourceTopicName          string `human:"Source Topic Name" serialized:"source_topic_name"`
-	MirrorStatus             string `human:"Mirror Status" serialized:"mirror_status"`
-	StatusTimeMs             int64  `human:"Status Time (ms)" serialized:"status_time_ms"`
-	Partition                int32  `human:"Partition" serialized:"partition"`
-	NumPartition             int32  `human:"Num Partition" serialized:"num_partition"`
-	PartitionMirrorLag       int64  `human:"Partition Mirror Lag" serialized:"partition_mirror_lag"`
-	MaxPerPartitionMirrorLag int64  `human:"Max Per Partition Mirror Lag" serialized:"max_per_partition_mirror_lag"`
-	LastSourceFetchOffset    int64  `human:"Last Source Fetch Offset" serialized:"last_source_fetch_offset"`
-	ErrorMessage             string `human:"Error Message" serialized:"error_message"`
-	ErrorCode                string `human:"Error Code" serialized:"error_code"`
+	LinkName                    string                     `human:"Link Name" serialized:"link_name"`
+	MirrorTopicName             string                     `human:"Mirror Topic Name" serialized:"mirror_topic_name"`
+	SourceTopicName             string                     `human:"Source Topic Name" serialized:"source_topic_name"`
+	MirrorStatus                string                     `human:"Mirror Status" serialized:"mirror_status"`
+	StatusTimeMs                int64                      `human:"Status Time (ms)" serialized:"status_time_ms"`
+	Partition                   int32                      `human:"Partition" serialized:"partition"`
+	NumPartition                int32                      `human:"Num Partition" serialized:"num_partition"`
+	PartitionMirrorLag          int64                      `human:"Partition Mirror Lag" serialized:"partition_mirror_lag"`
+	MaxPerPartitionMirrorLag    int64                      `human:"Max Per Partition Mirror Lag" serialized:"max_per_partition_mirror_lag"`
+	LastSourceFetchOffset       int64                      `human:"Last Source Fetch Offset" serialized:"last_source_fetch_offset"`
+	ErrorMessage                string                     `human:"Error Message" serialized:"error_message"`
+	ErrorCode                   string                     `human:"Error Code" serialized:"error_code"`
+	MirrorStateTransitionErrors []mirrorTransitionErrorOut `serialized:"mirror_state_transition_errors"`
```

Comment:
Why aren't we printing this to human output?

Reply:
We do but we have special logic to print a separate list so we don't duplicate the error info for each partition.
For example:
```
  Link Name | Mirror Topic Name | Source Topic Name | Mirror Status | Status Time (ms) | Partition | Partition Mirror Lag | Last Source Fetch Offset  
------------+-------------------+-------------------+---------------+------------------+-----------+----------------------+---------------------------
  link-1    | dest-topic-1      | src-topic-1       | ACTIVE        |        111111111 |         0 |               142857 |                  1293009  
  link-1    | dest-topic-1      | src-topic-1       | ACTIVE        |        111111111 |         1 |               285714 |                 28340404  
  link-1    | dest-topic-1      | src-topic-1       | ACTIVE        |        111111111 |         2 |               571428 |                  5739304  
  Mirror State Transition Error  | Mirror State Transition Error   
               Code              |            Message              
---------------------------------+---------------------------------
  AUTHENTICATION_ERROR           | Auth issue.                     
  MISCONFIGURATION_ERROR         | Wrong config.                   

```
We just print the error info once, instead of on each line for each partition.

I'm guessing you don't like having this separate table per your other comment. Need to think about this and syncing with my team.

Reply:
As with the other comment, I'll get input from @lukeknep here.

Reply:
@brianstrauch I've created a new command: `confluent kafka mirror describe-state-transition-errors` that is responsible for the state transition error info.

Reply:
Thanks! Mirroring the syntax of our other commands, this should be named `confluent kafka mirror state-transition-errors describe`. Bonus points if there's a way to write the subcommand name to use fewer dashes.